apiserver/facades/client/applicationoffers/service.go

```diff
@@ -99,3 +99,9 @@ type modelDomainServices struct {
 func (f *modelDomainServices) Application() ApplicationService {
 	return f.domainServices.Application()
 }
+
+type ModelService interface {
+	ListAllModels(ctx context.Context) ([]model.Model, error)
```

Comment:
Can you put some comments on these please? You are free to copy and paste the comments from the service :)

Reply:
done thanks tom!